diff -uraN owncloudclient-2.6.1.13407/src/libsync/networkjobs.cpp omv-owncloudclient-2.6.1.13407/src/libsync/networkjobs.cpp
--- owncloudclient-2.6.1.13407/src/libsync/networkjobs.cpp	2020-01-28 15:57:39.000000000 +0100
+++ omv-owncloudclient-2.6.1.13407/src/libsync/networkjobs.cpp	2020-06-12 13:54:27.056661100 +0200
@@ -29,6 +29,7 @@
 #include <QCoreApplication>
 #include <QJsonDocument>
 #include <QJsonObject>
+#include <QPainterPath>
 #ifndef TOKEN_AUTH_ONLY
 #include <QPainter>
 #endif
